opentelemetry/src/trace/mod.rs

```diff
@@ -200,6 +200,10 @@ pub enum TraceError {
     #[error("Exporting timed out after {} seconds", .0.as_secs())]
     ExportTimedOut(time::Duration),
 
+    /// already shutdown error
+    #[error("TracerProvider already shutdown")]
+    AlreadyShutdown,
```

Comment:
Should we rename this to be more specific?

```suggestion
    TracerProviderAlreadyShutdown,
```

Reply:
done.

Reply:
It doesn't show the change.

Reply:
oh missed pushing one of the files, done now.